packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -121,10 +121,9 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
     private readonly mc: MonitoringContext;
 
     /**
-     * Map of local (offline/detached) IDs to storage IDs. Contains identity entries
-     * (id → id) for storage IDs, so all requested IDs should be a key in this map.
-     * Blobs created while the container is detached are stored in IDetachedBlobStorage
-     * which gives local IDs; the storage IDs are filled in at attach time.
+     * Map of local IDs to storage IDs. Contains identity entries (id → id) for storage IDs. All requested IDs should
+     * be a key in this map. Blobs created while the container is detached are stored in IDetachedBlobStorage which
+     * gives local IDs; the storage IDs are filled in at attach time.
      */
```

Comment:
Maybe add a note "This table contains mappings for all attached blobs, regardless of the originating client".

Since "local ID" can be misleading like it only applies to the local client where the blob originated.